salt/apps/ocw/ocw_aka/install.sls

```diff
@@ -0,0 +1,35 @@
+{% set users = [fsuser, ocwuser, sshacs] %}
+{% set root_directory = '/ocwdata/apache-tomcat/webapps/' %}
+{% set directories = ['about', 'course', 'donate', 'educator', 'faculty', 'give',
+                     'help', 'high-school', 'icons', 'images', 'jsp', 'jw-player-free',
+                     'jwplayer', 'OcwWeb', 'ocw-labs', 'resources', 'rss', 'scp48112',
+                     'scripts', 'search', 'styles', 'subscribe', 'support', 'terms',
+                     'webfonts', 'xml', 'xsd'] %}
+
+{% for user in users %}
+create_{{ user }}_user:
+  - name: {{ user }}
+  - shell: /bin/bash
```

Comment:
We should probably populate the users with appropriate ssh_authorized_keys as well.

Reply:
@blarghmatey  This doesn't appear to be an option in the state. Did you mean just using some kinda `file.manage` to add the keys after creating the users?

Reply:
There's a module `ssh.set_auth_key` that can be wrapped in a state